exchanges/order/orders.go

```diff
@@ -476,6 +488,25 @@ func (d *Detail) Copy() Detail {
 	return c
 }
 
+// GetCopySlice returns a copy of all order details
+func GetCopySlice(old []*Detail) []Detail {
```

Comment:
We have plenty of places where we use `[]order.Detail` and this would require a conversion to a pointer list to copy the slice, then return a non-pointered slice. This seems like it should be like-for-like and become `(old []Detail) []Detail {`

Reply:
`CopyOrders`/`CopyOrderSlice`

Reply:
Yeah, I removed that completely because it's not being used 🤷 . I think in future I will have a type Details/Orders and have a copy method on that. But I think that's out of scope for this.